codespell_lib/tests/test_dictionary.py

```diff
@@ -65,21 +68,28 @@ def test_dictionary_formatting(fname, in_aspell):
 
 
 def _check_aspell(phrase, msg, in_aspell, fname):
-    if speller is None:
+    if not bool(spellers):  # if no spellcheckers exist
         return  # cannot check
     if in_aspell is None:
         return  # don't check
+    elif in_aspell is False:
+        language = supported_languages
+    elif in_aspell is True:
+        language = ('en',)
+    else:
+        language = in_aspell
     if ' ' in phrase:
         for word in phrase.split():
             _check_aspell(word, msg, in_aspell, fname)
         return  # stop normal checking as we've done each word above
-    this_in_aspell = speller.check(
-        phrase.encode(speller.ConfigKeys()['encoding'][1]))
-    end = 'be in aspell for dictionary %s' % (fname,)
-    if in_aspell:  # should be an error in aspell
-        assert this_in_aspell, '%s should %s' % (msg, end)
-    else:  # shouldn't be
+    this_in_aspell = any(spellers[lang].check(phrase.encode(
+        spellers[lang].ConfigKeys()['encoding'][1])) for lang in language)
+    end = 'be in aspell dictionaries %s for dictionary %s' % (
+        ' '.join(map(str, language)), fname,)
+    if not in_aspell:  # should be an error in aspell
```

Comment:
Were the previous comments wrong then? Might be better to just fix the comments rather than flip the logic of the `if`?

Reply:
no, i just didnt know, that you can do if variable and it is true, if the variable is set.

Reply:
In Python, `False`, `None`, `0` and `''` (empty string) evaluate to `False` in a boolean context, other values evaluate to `True`. More at https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/